decidim-core/db/seeds.rb

```diff
@@ -17,7 +17,7 @@
     end,
     homepage_image: File.new(File.join(__dir__, "seeds", "homepage_image.jpg")),
     default_locale: I18n.default_locale,
-    available_locales: Decidim.available_locales,
+    available_locales: [:en, :ca, :es],
```

Comment:
Hi @deivid-rodriguez ! I know this is merged but I am having a few problems because of that 😄 . Do you remember why you hardcoded this values? 
The problem I found is when a installation doesn't have English for example, I cannot create a review app because the seeds are creating "invalid" organizations.